### DISCUSSION TITLE
Docker Build

### DISCUSSION THREAD

**Discussion by Wraiith32** [lloesche/valheim-server-docker#42] [Q&A]
Is there anything special to actually build this docker file if I want to do it myself?

I pulled the code, ran "docker build ." which appears to be successful but when I run the server, it fails.

**Comment by InB4DevOps** [comment]
Try 
docker build -t valheim .

Then run the name of the supplied tag (valheim in this example)

**Reply by Wraiith32** [reply]
Unfortunately, that did not work. This is what the output shows:

![image](https://user-images.githubusercontent.com/26192516/108103822-51a5c100-703f-11eb-890b-4ef235fa8ae2.png)

**Reply by InB4DevOps** [reply]
What did you change after pulling the repo? ;-)

**Reply by Wraiith32** [reply]
I have not changed anything yet.  Wanted to do a local build before I did make any changes to ensure it works.
